packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -180,9 +180,17 @@ export class Button extends FormAssociatedButton {
     };
 
     /**
-     * References the root element
+     * @deprecated
      */
```

Comment:
@litteredwitherrors - looks like the deprecation block is missing a note so the build is broken. If we commit the below it should fix.

```suggestion
    /**
     * This has been deprecated in favor of control property
     * @deprecated
     */
```

Reply:
Sounds good. Commit away.

Reply:
Nevermind, looks like I'll have to push a change.